### ISSUE TITLE
React Warning for Typography.Paragraph: FunctionComponents cannot be given refs

### ISSUE THREAD

**Issue by MarcusCemes** [ant-design/ant-design#19088]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/embed/bold-mestorf-7ypmg)

### Steps to reproduce
1. Run the provided reproduction example
2. Check the console (may need to check browser console, not the sandbox console)

### What is expected?
No warnings

### What is actually happening?
The console prints:
```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `ResizeObserver`.
    in Typography (created by ResizeObserver)
    in ResizeObserver (created by LocaleReceiver)
    in LocaleReceiver (created by Base)
    in Base (created by Context.Consumer)
    in withConfigConsumer(Base) (created by Paragraph)
    in Paragraph (at src/index.js:6)
    in App (at src/index.js:10)
```

| Environment | Info |
|---|---|
| antd | 3.23.5 |
| React | 16.8.6 |
| System | Ubuntu 19 |
| Browser | Google Chrome |

---
The sandbox is a little bloated with errors, in a simple clean React app, any use of Typography.Paragraph will throw that error in development.

**Comment by StallionV** [comment]
Yes this has started showing up recently for us too

**Comment by erindru** [comment]
I am also getting this for Typography.Title - which lead me here. I wasted ~40mins thinking it was in my own code before realising it was being triggered by antd.

I had just done an update to 3.23.5 when this started occurring.

**Comment by atbtechgit** [comment]
Same here. I spent about 20 min thinking it was my code as well.

**Comment by vmedar** [comment]
Me too.

**Comment by afc163** [comment]
Fixed in 3.23.6
